docs/installation/steps/install-dependencies.md

```diff
@@ -0,0 +1,33 @@
+# Install related dependencies
+
+To enable the compilation, execution, and execution of complex neural network computations or models on our Virtual Machine using GPU acceleration, it is essential to establish the necessary foundational components. The initial stepS in this process involve installing TensorFlow and cunDNN, which serves as the foundational dependencies. Subsequently, our focus will shift towards the installation of supplementary libraries to augment this robust foundation.
```

Comment:
@maryalexa91 We need to mention that we will be using `pip` (not `conda`) for installing Python dependencies.

Reply:
Comments added

Reply:
Commit ff274a58ee96f061b99dc9ed598e4a93d12c71bd addressed this. Resolving this convo.